### DISCUSSION TITLE
Is it possible to change the serialization of NaiveDateTime on sqlite?

### DISCUSSION THREAD

**Discussion by remagpie** [diesel-rs/diesel#3390] [Q&A] [answered]
https://github.com/diesel-rs/diesel/blob/f560c29b4226d8651b6946de19e1cfcaea479309/diesel/src/sqlite/types/date_and_time/chrono.rs#L104

From the source code it seems that the serialization format of NaiveDateTime is fixed to `"%F %T%.f"` on sqlite.
I'm trying to use `%FT%T` format since my database already uses it, but I can't find any way of changing it.
Is it possible to change it?

**Comment by weiznich** [comment, marked as answer]
It's currently not possible to change this outside of diesel, at least not without writing a new type wrapper using custom serialization methods. The `#[diesel(serialize_as = OtherType)]` attribute of the [`Insertable`](https://docs.diesel.rs/master/diesel/prelude/derive.Insertable.html#optional-field-attributes and `AsChangeset` derive might be helpful here.
